fix_installation.sh

```diff
@@ -326,60 +451,33 @@ create_user_linux() {
 		"mysqldump"
 		"ldconfig"
 	)
-
-	SUDOERS_CONTENT="$username ALL=(ALL) ALL\n"
-	for cmd in "${iob_commands[@]}"; do
-		# Test each command if and where it is installed
-		cmd_bin=$(echo $cmd | cut -d ' ' -f1)
-		cmd_path=$(which $cmd_bin 2> /dev/null)
-		if [ $? -eq 0 ]; then
-			# Then add the command to SUDOERS_CONTENT
-			full_cmd=$(echo "$cmd" | sed -e "s|$cmd_bin|$cmd_path|")
-			SUDOERS_CONTENT+="$username ALL=(ALL) NOPASSWD: $full_cmd\n"
-		fi
-	done
+	add2sudoers "$username ALL=(ALL) " "${iob_commands[@]}"
 
 	# Additionally, define which iobroker-related commands may be executed by every user
 	declare -a all_user_commands=(
 		"systemctl start iobroker"
 		"systemctl stop iobroker"
 		"systemctl restart iobroker"
 	)
-	for cmd in "${all_user_commands[@]}"; do
-		# Test each command if and where it is installed
-		cmd_bin=$(echo $cmd | cut -d ' ' -f1)
-		cmd_path=$(which $cmd_bin 2> /dev/null)
-		if [ $? -eq 0 ]; then
-			# Then add the command to SUDOERS_CONTENT
-			full_cmd=$(echo "$cmd" | sed -e "s|$cmd_bin|$cmd_path|")
-			SUDOERS_CONTENT+="ALL ALL=NOPASSWD: $full_cmd\n"
-		fi
-	done
+	add2sudoers "ALL ALL=" "${all_user_commands[@]}"
 
 	# Furthermore, allow all users to execute node iobroker.js as iobroker
 	if [ "$IOB_USER" != "$USER" ]; then
-		cmd="node $CONTROLLER_DIR/iobroker.js"
-		cmd_bin=$(echo $cmd | cut -d ' ' -f1)
-		cmd_path=$(which $cmd_bin 2> /dev/null)
-		if [ $? -eq 0 ]; then
-			# Then add the command to SUDOERS_CONTENT
-			full_cmd=$(echo "$cmd" | sed -e "s|$cmd_bin|$cmd_path|")
-			SUDOERS_CONTENT+="ALL ALL=($IOB_USER) NOPASSWD: $full_cmd\n"
-		fi
+		add2sudoers "ALL ALL=($IOB_USER) " "node $CONTROLLER_DIR/iobroker.js"
 	fi
-	# TODO: ^ Can we reduce code repetition in these 3 blocks? ^
 
 	SUDOERS_FILE="/etc/sudoers.d/iobroker"
+	$SUDOX rm -f $SUDOERS_FILE
+# ADOE: ./temp_sudo_file   vs.   ~/temp_sudo_file
+# ADOE: caveat: FIXER <> INSTALLER!   why?
```

Comment:
I think we can always use `./temp_sudo_file` if that is what you meant. What about the 2nd comment, I don't see what you mean?

Reply:
1. OK. I thought to always use `~/temp_sudo_file` because we can not guarantee, where the user is currently when he uses `sudo su` without `-`.
2. I meant: in FIXER, root or not, it is always `~/`.
In INSTALLER, root has `./` and nonroot has  `~/`.

Reply:
You're right. I assumed this is always executed while we're already in `$IOB_DIR`. Doing it in ~ is fine IMO.